signature/openpgp/openpgp_test.go

```diff
@@ -0,0 +1,102 @@
+package openpgp
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/containers/image/signature"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestImportKeysFromBytes(t *testing.T) {
+	m, _ := NewOpenPGPSigningMechanism()
+	keyBlob, err := ioutil.ReadFile("../fixtures/public-key.gpg")
+	require.NoError(t, err)
+	keyIdentities, err := m.ImportKeysFromBytes(keyBlob)
+	require.NoError(t, err)
+	assert.Equal(t, []string{signature.TestKeyFingerprint}, keyIdentities)
+}
+
+func TestVerify(t *testing.T) {
+
+	type tc struct {
+		fixture      string
+		fixtureBytes []byte
+		isValid      bool
+		identity     string
+		publicKey    string
+	}
+
+	cases := []tc{
```

Comment:
I can add human description as messages in the table test, so when it spits error you can quickly identify the affected case.